orc8r/cloud/go/services/tenants/servicers/storage/storage.go

```diff
@@ -20,6 +20,8 @@ type Store interface {
 	GetAllTenants() (*protos.TenantList, error)
 	SetTenant(tenantID int64, tenant protos.Tenant) error
```

Comment:
implementation of `SetTenant` and `CreateTenant` are exactly the same. Prefer code reuse